qiskit/transpiler/synthesis/aqc/aqc_plugin.py

```diff
@@ -109,7 +109,7 @@ def run(self, unitary, **options):
         from qiskit.transpiler.synthesis.aqc.aqc import AQC
         from qiskit.transpiler.synthesis.aqc.cnot_structures import make_cnot_network
         from qiskit.transpiler.synthesis.aqc.cnot_unit_circuit import CNOTUnitCircuit
-        from qiskit.transpiler.synthesis.aqc.cnot_unit_objective import DefaultCNOTUnitObjective
+        from qiskit.transpiler.synthesis.aqc.fast_gradient.fast_gradient import FastCNOTUnitObjective
```

Comment:
```suggestion
        from qiskit.transpiler.synthesis.aqc.fast_gradient.fast_gradient import (
            FastCNOTUnitObjective,
        )
```

to fix black's line-length complaint